test/cadet/assessments_test.exs

```diff
@@ -0,0 +1,251 @@
+defmodule Cadet.AssessmentsTest do
```

Comment:
Use coveralls to track functions that has not been tested. You can also track locally by running `mix coveralls.html` and opening the generated report.

For example, there's missing coverage for `missions_due_soon/0`, `build_answer/1`, `change_question/2`, and so on.